sut/modules/unish/woot/drush.services.yml

```diff
@@ -1,11 +1,16 @@
 services:
+  logger.channel.woot:
+    class: Drupal\Core\Logger\LoggerChannel
+    factory: logger.factory:get
```

Comment:
`drush.services.yml` is deprecated in Drush 12. A legacy service instantiator is provided for backwards compatibility purposes;  it is functional enough to satisfy the needs of most Drush commands, but it is not a full-service DI container. It does not support the `factory` element, for example.

I would suggest trying to set this up using the `create` method. See `WootStaticFactoryCommands` as an example.